packages/web/src/pages/index.tsx

```diff
@@ -31,16 +31,16 @@ const Index = (_: Props) => {
   }, [router])
 
   return (
-    <>
+    <div style={{ background: 'white' }}>
```

Comment:
[nits] Does this background color setting need to be reflected on all pages?

Reply:
That certainly looks like unnecessary styles...

But I guess it's all good!

Reply:
The white background was chosen for a few reasons:

-Gives a modern aesthetic to the UI
-Maximum contrast with text 
-More freedom to use different colors

Here are other examples of crypto websites opting for white backgrounds:
[](https://ourzora.com)
[](https://opensea.io)

White offers a better canvas for layers with shadows, text, images with drop shadows which Stakes Social has plenty of.